### ISSUE TITLE
Play file from URL

### ISSUE THREAD

**Issue by nstbayless** [nstbayless/flyweb-mp3#10]
- User can select "From URL" in the add page (/p/add)
- User pastes a URL here.
- _if this is the URL for a music file_, the server does the following:
  - downloads the song from the URL
  - grabs its metadata to create the song object, adds it to the playlist
  - plays the music file when the song is reached in the queue
  - deletes the file afterward, but remembers the URL in case the song comes up again.
- _if this is the URL for a web page with media on it_, the server does something else (TBD in another issue)

**Comment by joshvazquez** [comment]
Proof of concept written. Need to resolve error with layout file before pushing.

**Comment by joshvazquez** [comment]
Also works for Vimeo links!

**Comment by joshvazquez** [comment]
Depends on #23 since YouTube seems to prefer 48 kHz stereo or mono.

**Comment by nstbayless** [comment]
You can use ffmpeg to convert to 44100: [stackexchange](http://superuser.com/a/253470)

**Comment by nstbayless** [comment]
#23 is now resolved, so this is not being blocked anymore.

**Comment by joshvazquez** [comment]
This was finished a while ago.
